Tools/TestWebKitAPI/Tests/WebKitCocoa/WKWebExtensionAPITabs.mm

```diff
@@ -1396,6 +1461,160 @@
     [manager loadAndRun];
 }
 
+TEST(WKWebExtensionAPITabs, ExecuteScript)
+{
+    TestWebKitAPI::HTTPServer server({
+        { "/"_s, { { { "Content-Type"_s, "text/html"_s } }, "<iframe src='/frame.html'><html><body onload=\"document.body.style.background ='blue'\"/></html></iframe>"_s } },
```

Comment:
iframes can't have inline content like this, it should just be:

`<iframe src='/frame.html'></iframe>`

And the `frame.html` should be:

`<body style='background-color: blue'></body>`